src/scss/includes/panels/poi_panel.scss

```diff
@@ -337,25 +337,14 @@ $HEADER_SIZE: 30px;
   float: left;
   text-decoration: none;
   text-align: center;
-  min-width: 45px;
-
-  &:not(:last-child){
-    margin-right: 15px;
-    max-width: calc(23% - 20px);
-  }
-
-  &.poi_panel__actions__phone_container {
-    max-width: 28%;
-  }
-
-  &:not(:first-child){
-    min-width: 20%
-  }
+  min-width: 24%;
```

Comment:
I'm a bit skeptical about those changes: why setting this value three times? Also: it'll need to be checked very carefully.